modules/extensions/scripts/kubeconfig_set_credentials.template.sh

```diff
@@ -2,7 +2,7 @@
 # Copyright 2021 Oracle Corporation and/or affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
-kubectl config set-credentials "user-${cluster-id-11}" --exec-command="./token_helper.sh" \
+kubectl config set-credentials "user-${cluster-id-11}" --exec-command="/usr/local/bin/token_helper.sh" \
```

Comment:
What process is installing the `token_helper.sh` script?

Reply:
Terraform is doing that.

Reply:
Does Terraform have a preferred location for helper scripts/binaries that's more local to the module? Having this in `/usr/local/bin` makes it available to _everyone_, which may be unexpected. It's possibly better placed somewhere within the module hierarchy and sourced using an absolute path.

Reply:
this will be invoked mainly by the opc user. so we could perhaps move to to /home/opc/bin?

Reply:
Either that or `/home/opc/.local/bin` perhaps.

Reply:
moved to $HOME/bin for opc user